Source/Models/ChainpointResponse/ChainpointError.swift

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
Please delete all header file comments. As we discussed in other forums, it useless information. Also, we do not care about copyright either.